scripts/Update-OnETagChanged.ps1

```diff
@@ -7,7 +7,9 @@ function Update-OnETagChanged() {
     [uri]$execUrl,
     [string]$saveFile = ".\info",
     [scriptblock]$OnETagChanged,
-    [scriptblock]$OnUpdated
+    [scriptblock]$OnUpdated,
+	  [string]$kind,
+	  [switch]$boule
```

Comment:
What is the intention of this switch?
Calling it `boule` doesn't say anything about what it is for, nor am I familiar with the word and seem to me you have taken the word from a different language.